diff -ruN lv-mod-4.51.a-007.orig/src/conf.c lv-mod-4.51.a-007/src/conf.c
--- lv-mod-4.51.a-007.orig/src/conf.c	2014-12-13 03:13:59.000000000 +0900
+++ lv-mod-4.51.a-007/src/conf.c	2022-10-28 01:06:45.000000000 +0900
@@ -84,6 +84,7 @@
 /*#define LV_HELP_PATH	"/usr/local/lib/lv/"*/ /* now defined through make */
 
 private byte *lvHelpFile[ 2 ];
+boolean_t stream_filter;
 
 private void ConfInitArgs( conf_t *conf )
 {
@@ -486,8 +487,18 @@
     strcat( helpFile, LV_HELP );
   }
 #else
-  helpFile = Malloc( strlen( LV_HELP_PATH "/" LV_HELP ) + 1 );
-  strcpy( helpFile, LV_HELP_PATH "/" LV_HELP );
+  {
+    char *p = getenv( "LV_HELP_PATH" );
+    if(p){
+      helpFile = Malloc( strlen( p ) + strlen( "/" LV_HELP ) + 1 );
+      strcpy( helpFile, p );
+      strcat( helpFile, "/" LV_HELP );
+    }
+    else{
+      helpFile = Malloc( strlen( LV_HELP_PATH "/" LV_HELP ) + 1 );
+      strcpy( helpFile, LV_HELP_PATH "/" LV_HELP );
+    }
+  }
 #endif /* MSDOS || WIN32NATIVE */
 }
 
diff -ruN lv-mod-4.51.a-007.orig/src/stream.h lv-mod-4.51.a-007/src/stream.h
--- lv-mod-4.51.a-007.orig/src/stream.h	2014-12-13 03:13:59.000000000 +0900
+++ lv-mod-4.51.a-007/src/stream.h	2022-10-28 01:03:23.000000000 +0900
@@ -16,7 +16,7 @@
   int  pid;
 } stream_t;
 
-public boolean_t stream_filter;
+extern boolean_t stream_filter;
 
 public stream_t *StreamOpen( byte *file );
 public boolean_t StreamClose( stream_t *st );
